src/main/java/world/bentobox/bentobox/api/configuration/WorldSettings.java

```diff
@@ -467,4 +467,61 @@ default boolean isTeleportPlayerToIslandUponIslandCreation() {
         return true;
     }
 
+
+    /**
+     * Returns all aliases for main admin command.
+     * It is assumed that all aliases are split with whitespace between them.
+     * String cannot be empty.
+     * Default value: {@code getFriendlyName() + "admin"} (to retain backward compatibility).
+     * @return String value
+     * @since 1.13.0
+     */
+    default String getAdminCommandAlias()
```

Comment:
```suggestion
    default String getAdminCommandAliases()
```

Just being a bit nit-picky here, but I think that the suggested name would be better for the future.